### PR TITLE
refactor(condo): HOTFIX use legal documents from env, instead of local files

### PR DESCRIPTION
**Pull request by toplenboren**
## Problem

Condo is open-source platform, but we still store some .pdf from the mother company. This should be fixed

## Solution

Move legal documents to the environment

### CODE REVIEW

**Comment by tolmachev21** [comment]
Should the dev portal and callcenter pointers be updated here?

**Comment by toplenboren** [comment]
> Should the dev portal and callcenter pointers be updated here?

Devportal is not a submodule and is updated here. cc does not have signup feature

**Comment by tolmachev21** [comment]
Screenshot 2024-10-07 at 15 54 36

In this page we not delete reCAPTCHA?

File: apps/condo/domains/user/components/CookieAgreement.tsx [hunk 1]
```diff
@@ -88,7 +92,12 @@ export const CookieAgreement: React.FC = () => {
                     </Space>
                     {displayMore &&
                         <Typography.Paragraph>
-                            <Typography.Text type='secondary' size='medium'>{CookieAgreementDescriptionMsg}</Typography.Text>
+                            <Typography.Text type='secondary' size='medium'>{CookieAgreementDescriptionMsg} </Typography.Text>
+                            { privacyPolicyUrl && (
+                                <Typography.Text type='secondary' size='medium'>
+                                    {CookieExpandActionMsg} <Link href={privacyPolicyUrl}>{privacyPolicyUrl}</Link>
+                                </Typography.Text>
```
Comment (Alllex202) [inline apps/condo/domains/user/components/CookieAgreement.tsx:99]: Will this open in a new tab?

**Review by Alllex202 — APPROVED** [review]

**Comment by Alllex202** [comment]
> In this page we not delete reCAPTCHA?

By the way, we haven't used `reCAPTCHA` for a long time. We should delete it

**Review by tolmachev21 — APPROVED** [review]

**Pull request merged**